src/electron/add_tap_device.bat

```diff
@@ -75,6 +75,13 @@ if %errorlevel% neq 0 (
 
 echo Configuring new TAP network device...
 
+:: check if newly created TAP adapter is ready for netsh
```

Comment:
Great. So that we don't force a `sleep` (the installer is slow enough) how about:

```bat
...
...
echo Searching for new TAP network device name...
powershell "(compare-object (cat \"%BEFORE_DEVICES%\").trim() (cat \"%AFTER_DEVICES%\").trim() | format-wide -autosize | out-string).trim() | set-variable NEW_DEVICE; write-host \"New TAP device name: ${NEW_DEVICE}\"; netsh interface set interface name = \"${NEW_DEVICE}\" newname = \"%DEVICE_NAME%\"" <nul
if %errorlevel% neq 0 (
  echo Could not find or rename new TAP network device. >&2
  exit /b 1
)

:: We've occasionally seen delays before netsh will "see" the new device, at least for
:: purposes of configuring IP and DNS ("netsh interface show interface name=xxx" does not
:: seem to be affected).
echo Testing that the new TAP network device is visible to netsh...
netsh interface ip show interfaces | find "%DEVICE_NAME%"
if errorlevel 0 goto :configure

:loop
echo waiting...
sleep 1
netsh interface ip show interfaces | find "%DEVICE_NAME%"
if errorlevel 1 goto :loop

:configure
echo Configuring new TAP network device...
...
...
```